src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/runByPowerUntilCondition/ForwardRunFunnelUntilObjectIn.java

```diff
@@ -0,0 +1,12 @@
+package edu.greenblitz.robotName.commands.shooter.funnel.runByPowerUntilCondition;
+
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.Funnel;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.FunnelConstants;
+
+public class ForwardRunFunnelUntilObjectIn extends RunFunnelByPowerUntilCondition {
+
+    public ForwardRunFunnelUntilObjectIn() {
+        super(FunnelConstants.ROLL_POWER, () -> Funnel.getInstance().isObjectIn());
```

Comment:
Use “funnel” variable of from FunnelCommand

Reply:
You cant because `funnel` is null